### DISCUSSION TITLE
Young modulus variable depending upon

### DISCUSSION THREAD

**Discussion by AlejandroRiano2023** [idaholab/moose#28233] [Q&A Modules: Phase field]
Hello, I hope you are doing great!

Usually, the Young Modulus is defined as a constant.

    [./elasticity_tensor]
    type = ComputeIsotropicElasticityTensor
    youngs_modulus = 200
    poissons_ratio = 0.3
    [../]

Is there a way to define Young Modulus that can depend on arbitrary variables I created?

I will be attentive to your comments and suggestions and thank you so much in advance.

Best regards,
Alejandro

**Comment by GiudGiud** [comment]
Hello

There is this one for example which will let you use variables instead of constants
https://mooseframework.inl.gov/source/materials/ComputeVariableIsotropicElasticityTensor.html

Guillaume
